lib/takeaway.rb

```diff
@@ -0,0 +1,47 @@
+class Takeaway
+  attr_reader :basket
+
```

Comment:
How about this refactor?

```ruby
def initialize(menu)
  @menu = menu
  @basket = {}
  @total_cost = 0
  @basket_with_cost = {}
end
```

It would allow you to change the menu easily when creating new Takeaways